lighthouse-core/audits/byte-efficiency/unminified-javascript.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const ByteEfficiencyAudit = require('./byte-efficiency-audit');
+const esprima = require('esprima');
+
+const IGNORE_THRESHOLD_IN_PERCENT = .1;
+const IGNORE_THRESHOLD_IN_BYTES = 2048;
+
+class UnminifiedJavaScript extends ByteEfficiencyAudit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      name: 'unminified-javascript',
+      description: 'Minify JavaScript',
+      informative: true,
+      helpText: 'Minifying JavaScript files can reduce payload sizes and script parse time.' +
+        '[Learn more](https://developers.google.com/speed/docs/insights/MinifyResources).',
+      requiredArtifacts: ['Scripts', 'devtoolsLogs'],
+    };
+  }
+
+  /**
```

Comment:
can you add a comment here explaining the basic approach?  using the description from this PR works for me.

let's also call out that inline scripts are not evaluated. (i dont think it matters in practice, but since other audits of ours include inline scripts we should just be explicit)

Reply:
yeah sounds good 👍 

done